data/json/recipes/food/frozen.json

```diff
@@ -0,0 +1,247 @@
+[
+  {
+    "result": "milkshake",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 1,
+    "time": 10000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "components": [
+      [ [ "milk", 1 ], [ "con_milk", 1 ], [ "can_coconut", 1 ] ],
+      [
+        [ "chocolate", 1 ],
+        [ "candy", 1 ],
+        [ "candy2", 1 ],
+        [ "candy3", 1 ],
+        [ "maple_candy", 1 ],
+        [ "syrup", 1 ],
+        [ "strawberries", 1 ],
+        [ "strawberries_cooked", 1 ],
+        [ "irradiated_strawberries", 1 ],
+        [ "blueberries", 1 ],
+        [ "blueberries_cooked", 1 ],
+        [ "irradiated_blueberries", 1 ],
+        [ "blackberries", 1 ],
+        [ "irradiated_blackberries", 1 ],
+        [ "cherries", 1 ],
+        [ "irradiated_cherries", 1 ]
+      ],
+      [ [ "sugar", 10 ] ],
+      [ [ "water_clean", 1 ] ]
+    ]
+  },
+  {
+    "result": "milkshake_deluxe",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "time": 5000,
+    "autolearn": true,
+    "components": [
+      [ [ "milk", 1 ], [ "con_milk", 1 ], [ "can_coconut", 1 ] ],
+      [
+        [ "chocolate", 2 ],
+        [ "candy", 2 ],
+        [ "candy2", 2 ],
+        [ "candy3", 2 ],
+        [ "maple_candy", 2 ],
+        [ "syrup", 2 ],
+        [ "strawberries", 2 ],
+        [ "strawberries_cooked", 2 ],
+        [ "irradiated_strawberries", 2 ],
+        [ "blueberries", 2 ],
+        [ "blueberries_cooked", 2 ],
+        [ "irradiated_blueberries", 2 ],
+        [ "blackberries", 2 ],
+        [ "irradiated_blackberries", 2 ],
+        [ "cherries", 2 ],
+        [ "irradiated_cherries", 2 ]
+      ],
+      [ [ "sugar", 20 ] ],
+      [ [ "water_clean", 1 ] ]
+    ]
+  },
+  {
+    "result": "icecream",
+    "type": "recipe",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": 20000,
+    "autolearn": true,
+    "qualities": [ { "id": "COOK", "level": 3 } ],
+    "components": [
+      [ [ "milk", 2 ], [ "con_milk", 2 ], [ "can_coconut", 2 ] ],
```

Comment:
Most no-machine recipes use heavy cream + condensed milk and require a bit of churning. I'd add food processor, quern, or mortar and pestle to the tools and change the ingredients to milk, 2 and con_milk 2 OR can_coconut 2